utils/data_handlers.py

```diff
@@ -21,3 +21,16 @@ def save_typo_data(path, data):
             dumps({word: list(incorrect_words) for word, incorrect_words in data.items()},
                   indent=4,
                   sort_keys=True))
+
+def save_stats_file(path, data):
+    """Save statistics about typos."""
+    with open(path, "w") as f:
+        f.write(
+            dumps({wrong_word: counter for wrong_word, counter in data.items()},
+                  indent=4,
+                  sort_keys=True))
+
+def open_stats_file(path):
+    """Open a file with statistics on wrong words and return the data as a defaultdict of sets."""
```

Comment:
there is no check if the file exist

Reply:
yes you are right, if GUI starts and the file doesn't exist there is an exception.
I'll try to cover this scenario